src/utils/utils.cairo

```diff
@@ -236,6 +243,29 @@ namespace Helpers {
         return (bytes_array_len=32, bytes_array=value_as_bytes_array);
     }
 
+    // @notice This function is a variant of `uint256_to_bytes_array` that encodes the uint256 with no padding
+    // @param value: value to convert.
```

Comment:
convention question:

What is the preferred format for these document comments? In some places it looks like:

https://github.com/sayajin-labs/kakarot/blob/main/src/kakarot/stack.cairo#L55

and here I went with the flow and used colons.

Reply:
we are supposed to use natspec https://docs.soliditylang.org/en/v0.8.17/natspec-format.html

could you create an issue for cleaning the codebase and adding a tool like https://github.com/onlydustxyz/kaaper to generate the doc from the comment?